megamek/src/megamek/client/ui/swing/ClientGUI.java

```diff
@@ -1430,13 +1478,69 @@ private void toggleAccessibilityWindow() {
 
     private void toggleUnitOverview() {
         uo.setVisible(!uo.isVisible());
-        GUIPreferences.getInstance().setShowUnitOverview(uo.isVisible());
+        GUIP.setShowUnitOverview(uo.isVisible());
         bv.refreshDisplayables();
     }
 
     /** Shows or hides the minimap based on the current menu setting. */
     private void maybeShowMinimap() {
-        setMapVisible(GUIP.getMinimapEnabled());
+        GamePhase phase = getClient().getGame().getPhase();
+
+        if (phase.isReport()) {
+            String action = GUIP.getMinimapAutoDisplayReportPhase();
```

Comment:
This confuses me. Shouldn't a GUIP call return a boolean for something like this?